### PR TITLE
fix(keeper): validate tx queue env integers

### PR DESCRIPTION
**Pull request by Bayyan16**
## Summary

This PR hardens TxQueue environment parsing so queue configuration values only accept valid positive integers before being passed into `PQueue`.

Previously, `parseEnvInt()` accepted any numeric value returned by `parseInt()`, including invalid queue settings such as:

* `0`
* negative numbers like `-1`
* fractional-looking strings that could be partially parsed

Those values could then be used to construct queue lanes for liquidation, oracle, and crank work. In particular, invalid non-positive values such as `TX_QUEUE_CRANK_INTERVAL_CAP=-1` can cause `PQueue` construction to throw and prevent the keeper from initializing cleanly.

## Changes

* Update `parseEnvInt()` in `src/lib/tx-queue.ts` to:

  * trim environment input
  * fall back when the value is empty
  * parse with `Number()` instead of `parseInt()`
  * require `Number.isInteger(n)`
  * require `n > 0`
  * fall back to the safe default for invalid values

* Add regression coverage in `tests/lib/tx-queue-env-validation.poc.test.ts` proving that invalid non-positive `TX_QUEUE_*` values no longer construct an invalid queue.

## Why

TxQueue configuration comes from environment variables. If those values are accidentally set to invalid non-positive values, the keeper should not crash during queue construction. It should instead fall back to safe defaults.

This makes queue initialization more robust against misconfiguration and prevents invalid environment values from breaking keeper startup.

## Before

Invalid values such as:

```bash
TX_QUEUE_CRANK_CONCURRENCY=0
TX_QUEUE_CRANK_INTERVAL_CAP=-1
TX_QUEUE_CRANK_INTERVAL_MS=0
```

could be accepted by `parseEnvInt()` and passed into `PQueue`.

The regression test reproduced the failure:

```txt
TypeError: Expected `intervalCap` to be a number from 1 and up, got `-1` (number)
```

## After

Invalid non-positive values are rejected by `parseEnvInt()` and replaced with the configured fallback defaults.

The queue can initialize and process work normally.

## Testing

Passed:

```bash
pnpm exec vitest run tests/lib/tx-queue-env-validation.poc.test.ts --reporter=verbose
```

Result:

```txt
Test Files  1 passed (1)
Tests       1 passed (1)
```

## Notes

This PR is intentionally small and focused. It does not change queue behavior for valid environment values. It only hardens invalid environment parsing and adds regression coverage for the failure case.

## Release Notes

* **Bug Fixes**
  * Improved queue configuration validation to strictly accept only positive integer values from environment variables, with proper fallback handling for missing or invalid inputs.

* **Tests**
  * Added test coverage for queue configuration fallback behavior with invalid environment values.

### CODE REVIEW

File: src/lib/tx-queue.ts [hunk 1]
```diff
@@ -55,10 +55,11 @@ export interface TxLaneStats {
...
 function parseEnvInt(name: string, fallback: number): number {
-  const raw = process.env[name];
-  if (raw === undefined) return fallback;
-  const n = parseInt(raw, 10);
-  return isNaN(n) ? fallback : n;
+  const raw = process.env[name]?.trim();
+  if (!raw) return fallback;
+
+  const n = Number(raw);
+  return Number.isInteger(n) && n > 0 ? n : fallback;
```
[the remark this answers is not in the document]
Comment (Bayyan16) [reply src/lib/tx-queue.ts]: Updated the env parser to use per-setting minimums: concurrency and intervalCap require >= 1, while *_INTERVAL_MS accepts >= 0. I also updated the regression test wording to reflect that zero-ms intervals are valid.
